concordium-sdk/src/main/java/com/concordium/sdk/crypto/CryptoJniNative.java

```diff
@@ -316,4 +318,14 @@ public static String getVerifiableCredentialBackupEncryptionKey(String seedAsHex
      * {@link Web3IdProof} as JSON.
      */
     public static native String createWeb3IdProof(String input);
+
+    /**
+     * Check that a request is acceptable
+     * @param input {@link Request} serialized as JSON.
+     * @return JSON representing {@link StringResult}. If successful the field 'result' is empty, 
```

Comment:
You are kind of abusing `StringResult` if this is what you are doing. You could make a type specifically for this.